storm-core/src/ui/public/flux.html

```diff
@@ -0,0 +1,139 @@
+<html>
```

Comment:
No it shouldn't be only for Ambari. I'll see where's good location to put. Welcome any ideas.